src/View/batchprocessing/ROINameCleaningOptions.py

```diff
@@ -175,46 +171,87 @@ def get_standard_names(self):
                 self.volume_prefixes.append(row[1])
             f.close()
 
-    def create_table_view(self):
+    def create_table_view_organ(self):
         """
         Create a table to display all of the non-standard ROIs and
         options for what to do with them.
         """
         # Create table
-        self.table_roi = QtWidgets.QTableWidget(self)
-        self.table_roi.setStyleSheet(
+        self.table_organ = QtWidgets.QTableWidget(self)
+        self.table_organ.setStyleSheet(
             "background-color: rgb(255, 255, 255);")
-        self.table_roi.setColumnCount(4)
-        self.table_roi.verticalHeader().hide()
+        self.table_organ.setColumnCount(4)
+        self.table_organ.verticalHeader().hide()
         # Note - "New Name" is only enabled if the option "Rename" is
         # selected.
-        self.table_roi.setHorizontalHeaderLabels(
-            [" ROI Name ", " Option ", " New Name ", " Dataset Location "])
+        self.table_organ.setHorizontalHeaderLabels(
+            [" Organ Name ", " Option ", " New Name ", " Dataset Location "])
 
         # Set text align
-        self.table_roi.horizontalHeaderItem(0).setTextAlignment(
+        self.table_organ.horizontalHeaderItem(0).setTextAlignment(
             QtCore.Qt.AlignLeft)
-        self.table_roi.horizontalHeaderItem(1).setTextAlignment(
+        self.table_organ.horizontalHeaderItem(1).setTextAlignment(
             QtCore.Qt.AlignLeft)
-        self.table_roi.horizontalHeaderItem(2).setTextAlignment(
+        self.table_organ.horizontalHeaderItem(2).setTextAlignment(
             QtCore.Qt.AlignLeft)
-        self.table_roi.horizontalHeaderItem(3).setTextAlignment(
+        self.table_organ.horizontalHeaderItem(3).setTextAlignment(
             QtCore.Qt.AlignLeft)
 
         # Set header stretch
-        roi_name_header = self.table_roi.horizontalHeader()
+        roi_name_header = self.table_organ.horizontalHeader()
         roi_name_header.setSectionResizeMode(0, QtWidgets.QHeaderView.Stretch)
         roi_name_header.setSectionResizeMode(1, QtWidgets.QHeaderView.Stretch)
         roi_name_header.setSectionResizeMode(2, QtWidgets.QHeaderView.Stretch)
         roi_name_header.setSectionResizeMode(3, QtWidgets.QHeaderView.Stretch)
 
         # Removing the ability to edit tables with immediate click
-        self.table_roi.setEditTriggers(
+        self.table_organ.setEditTriggers(
             QtWidgets.QTreeView.NoEditTriggers |
             QtWidgets.QTreeView.NoEditTriggers)
 
         # Add table to the main layout
-        self.main_layout.addWidget(self.table_roi)
+        self.main_layout.addWidget(self.table_organ)
+
+    def create_table_view_volume(self):
+        """
+        Create a table to display all of the non-standard ROIs and
+        options for what to do with them.
+        """
+        # Create table
```

Comment:
Needs an info log here to specify that the function has started

Reply:
Thanks Ryan that is well pointed out.

Reply:
Fixed